source/routes/create/daily.js

```diff
@@ -0,0 +1,49 @@
+const PouchDB = require('pouchdb');
+PouchDB.plugin(require('pouchdb-find'));
+const db = new PouchDB('db');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/cse110-sp21-group13/cse110-sp21-group13/pullRequest?prid=7433333)
```suggestion
const db = new PouchDB("db");
```